### PR TITLE
Create statement data types fix

### PR DESCRIPTION
**Pull request by jcespinoza**
These commits add support for the remaining datatypes and also removes autogenerated files from version control

### CODE REVIEW

**Review by andresrgz — CHANGES_REQUESTED** [review]
These files shouldn't be ignored because the generated code is needed by the other modules.

**Comment by jcespinoza** [comment]
Damn it, you're right. Good call.

Sent from mobile

Em 20/11/2017 21:06, Andrés Rodríguez <notifications@github.com> escreveu:

@astrothing requested changes on this pull request.

These files shouldn't be ignored because the generated code is needed by the other modules.

File: types.go [hunk 1]
```diff
@@ -5,6 +5,7 @@ const (
 	charSize    = 1
 	integerSize = 4
 	floatSize   = 8
+	datetimeSize = 8
```
Comment (andresrgz) [inline types.go]: Change datetimeSize to 4.

**Review by andresrgz — CHANGES_REQUESTED** [review]

File: types.go (hunk 1 above)
Comment (andresrgz) [reply types.go]: Also floatSize to 4.

**Comment by jcespinoza** [comment]
@astrothing I already unignored the files and corrected the datatype sizes as per your comments

**Pull request merged**